### DISCUSSION TITLE
Troubleshooting login

### DISCUSSION THREAD

**Discussion by BoDonkey** [znck/grammarly#296] [Q&A]
Hi,
I'm using Grammarly desktop with a premium subscription. When I connect to Grammarly through the command palette, it opens a browser window and seemingly connects. However, the icon at the bottom turns red and states that there is a connection error. There isn't anything special in the output section. Any pointers on troubleshooting this?
Thanks

**Comment by lxy1993** [comment]
Hi,
I have the same problem！ Even it can not open a browser.

**Comment by trebeljahr** [comment]
to me, it seems like the Netlify redirect page is broken. 

at least I get an error from there. 
![image](https://user-images.githubusercontent.com/25333032/186937857-4a9ba5c6-7db7-4d9c-b0ab-eb01de99b918.png)

**Reply by aldinuc** [reply]
Same here

**Comment by znck** [comment]
Checking https://github.com/znck/grammarly/issues/315

**Comment by fxstein** [comment]
I have the same issue as of today. The login workflow seemingly works and tells me the account is connected. (So no issues with URLs) but the moment VSCode tries to log in, the dreaded read account symbol and exclamation mark are shown in the bottom corner. 

For info I have 2FA authentication enabled for my Grammarly account. Sure hope this is not causing this issue.

**Reply by znck** [reply]
There's a known issue when multiple instances of VS Code are running. Make sure you have only one active instance of VS Code when connecting your Grammarly account.

**Reply by fxstein** [reply]
Sorry, that's not it. Confirmed single instance of VSCode running. Restarted and updated everything. Uninstalled Grammarly Extension and reinstalled it, followed the account login workflow.

Get to the point where VSCode tells me account connected, but the icons turn red.

![Screen Shot 2022-10-23 at 2 02 19 PM](https://user-images.githubusercontent.com/773967/197418238-2f1f3b90-b12b-440b-8f84-07f4633a87f8.png)

Interestingly the workflow on the Grammarly website never seems to finish.
![Screen Shot 2022-10-23 at 2 06 04 PM](https://user-images.githubusercontent.com/773967/197418305-54de1fa4-84e4-4989-ab07-602177973d51.png)
Never gets past the spinning wheel.

Tried with both Safari and Chrome.

**Reply by fxstein** [reply]
Here is the VSCode version data:

```
Version: 1.72.2
Commit: d045a5eda657f4d7b676dedbfa7aab8207f8a075
Date: 2022-10-12T22:15:55.763Z
Electron: 19.0.17
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin arm64 21.6.0
Sandboxed: No
```

**Reply by fxstein** [reply]
FYI I switched to the most current pre-release version and everything started working. Will keep you updated.
